projects/simple_mlp/datapreprocessor.py

```diff
@@ -18,5 +38,11 @@
 #If you normalize the label, don't forget to unnormalize it in the prediction.
 print(X.shape, y.shape)
 print(y)
+

```

Comment:
Can you add all the preprocessing as a function? Or move it to the __name__="__main__"? As it is now, the torch.save will be launched every time you import simple_mlp.datapreprocessor